cmd/agent/app/throttler/config.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2018 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package throttling
+
+import (
+	"time"
+)
+
+// AccountConfig provides values to be used with an account object.
```

Comment:
Do we have a directory for that sort of thing?

Reply:
we have precedent of simply having README in the package, you can start there, but you can also create a top level `adr` dir. The difference is that the README would be expected to stay up to date, while the ADRs are just historical records of why certain things were built in a certain way, discussions of pros/cons, and decisions made, in fact. I would say the general architecture diagram should not be in the ADRs, since it's support to stay up to date, so I would just write a README for now.